packages/camera/tizen/src/camera_device.cc

```diff
@@ -345,20 +346,19 @@ CameraDevice::CameraDevice(flutter::PluginRegistrar *registrar,
               current_packet_ = nullptr;
               return nullptr;
             }
-            flutter_desktop_gpu_buffer_->buffer = surface;
-            flutter_desktop_gpu_buffer_->width = width;
-            flutter_desktop_gpu_buffer_->height = height;
-            flutter_desktop_gpu_buffer_->release_callback =
-                [](void *release_context) {
-                  CameraDevice *cd = (CameraDevice *)release_context;
-                  cd->ReleaseMediaPacket();
-                };
-            flutter_desktop_gpu_buffer_->release_context = this;
-            return flutter_desktop_gpu_buffer_.get();
+            gpu_surface_->handle = surface;
+            gpu_surface_->width = width;
+            gpu_surface_->height = height;
+            gpu_surface_->release_callback = [](void *release_context) {
+              CameraDevice *cd = (CameraDevice *)release_context;
```

Comment:
We may need another PR to do some refactoring (to fix the C type cast).

Reply:
I also think other PR is better.